e2e/steps_file.js

```diff
@@ -332,6 +332,10 @@ module.exports = function () {
       return caseId;
     },
 
+    async grantCaseAccess(caseId, user, role) {
+      await caseHelper.grantCaseAccess(caseId, user.email, role);
+    },
+
```

Comment:
is it needed for this PR? (minor, could be separated if not needed)

Reply:
yes, it is needed for this PR. 
in e2e i grant [SOLICITOR] role to user via api and then i check that this user cannot see confidential documents

Reply:
it is manageDocuments_test.js line 16